qiskit/circuit/library/pauli_evolution.py

```diff
@@ -152,6 +158,94 @@ def time(self, time: ParameterValueType) -> None:
         """
         self.params = [time]
 
+    def inverse(self, annotated: bool = False) -> Gate:
+        """Invert this instruction by negating the time.
+
+        Args:
+            annotated: If set to ``True`` the output inverse gate will be returned
+                as :class:`.AnnotatedOperation`.
+
+        Returns:
+            The inverse operation.
+        """
+        if annotated:
+            return AnnotatedOperation(self, InverseModifier())
+
+        return PauliEvolutionGate(self.operator, -self.time, synthesis=self.synthesis)
+
+    def power(self, exponent: float, annotated: bool = False) -> Gate:
+        """Raise this gate to the power of ``exponent``.
+
+        Implemented by multiplying the time by the exponent.
+
+        Args:
+            exponent: The power to raise the gate to.
+            annotated: indicates whether the power gate can be implemented
+                as an annotated operation.
+
+        Returns:
+            An operation implementing ``gate^exponent``
+        """
+        if not annotated:
+            return PauliEvolutionGate(self.operator, self.time * exponent, synthesis=self.synthesis)
```

Comment:
Similar comment to `inverse` here about whether `evo.power().synthesize()` is going to be equivalent to `evo.synthesize().power()`?

Reply:
No it won't be, we need to document this clearly 🤔